packages/web-app-files/src/mixins/deleteResources.js

```diff
@@ -6,7 +6,8 @@ import PQueue from 'p-queue'
 export default {
   data: () => ({
     deleteResources_queue: new PQueue({ concurrency: 4 }),
-    deleteResources_deleteOps: []
+    deleteResources_deleteOps: [],
+    filesToDelete: []
```

Comment:
Please use `resourcesToDelete` as variable name. While `resources` doesn't seem to be good wording for the rendered content, it is a good technical term in the code when referring to files & folders in general.